src-docs/src/views/theme/other/_levels_js.tsx

```diff
@@ -18,11 +23,114 @@ export default () => {
 
   return (
     <>
+      <ThemeExample
+        title="euiTheme.levels[type]"
+        description={
+          <>
+            <p>
+              Most of the time the z-index is handled per component. But you can
+              use the table below as a reference to how EUI layers the
+              components.
+            </p>
+          </>
+        }
+        examplePanel={{
+          color: 'subdued',
+        }}
+        example={
+          <EuiPanel
+            css={css`
+              width: 100%;
+              height: 200px;
+              position: relative;
+            `}
+          >
```

Comment:
Can you reduce the layering a bit by having this first one be the modal and shrink it in height some too? Maybe just a single toast? It's just a very complex example.